AdminWebsite/AdminWebsite/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -5293,7 +5293,8 @@ export class AddNewJusticeUserRequest implements IAddNewJusticeUserRequest {
     username?: string | undefined;
     /** The user's telephone */
     contact_telephone?: string | undefined;
-    role?: JusticeUserRole;
+    /** The user's role. This can be a VHO or a Team Lead. */
+    role?: JusticeUserRole[] | undefined;
```

Comment:
This looks like it needs updating since `role` is now `roles`

Reply:
done